docs-translations/zh-CN/tutorial/online-offline-events.md

```diff
@@ -3,15 +3,15 @@
 
 *main.js*
 ```javascript
-const electron = require('electron');
-const app = electron.app;
-const BrowserWindow = electron.BrowserWindow;
+const electron = require('electron')
+const app = electron.app
+const BrowserWindow = electron.BrowserWindow
 
-var onlineStatusWindow;
-app.on('ready', function() {
-  onlineStatusWindow = new BrowserWindow({ width: 0, height: 0, show: false });
-  onlineStatusWindow.loadURL('file://' + __dirname + '/online-status.html');
-});
+var onlineStatusWindow
+app.on('ready', function () {
```

Comment:
Fixed the above instance. I updated some snippets to ES2015 here and there, and in many cases copy-pasted snippets for the up-to-date English docs, but didn't make it a priority for this PR to update all the syntax.